test/integration/util/util.go

```diff
@@ -238,41 +239,56 @@ func (m *MinishiftRunner) CheckStatus(desired string) bool {
 }
 
 func (m *MinishiftRunner) GetProfileStatus(profileName string) string {
-	cmdOut, _, _ := m.RunCommand("--profile " + profileName + " status")
+	cmdOut, _, _, _ := m.RunCommand("--profile " + profileName + " status")
 	return strings.Trim(cmdOut, " \n")
 }
 
 func (m *MinishiftRunner) GetProfileList() string {
-	cmdOut, _, _ := m.RunCommand("profile list")
+	cmdOut, _, _, _ := m.RunCommand("profile list")
 	return strings.Trim(cmdOut, " \n")
 }
 
-func (m *MinishiftRunner) GetOpenshiftContainers(containerName string) (string, error) {
-	cmdOut, cmdErr, _ := m.RunCommand(fmt.Sprintf(`ssh -- docker ps -f "name=%s"`, containerName))
+func (m *MinishiftRunner) GetContainerID(containerName string) (string, error) {
+	cmd := fmt.Sprintf(`ssh -- docker ps -a -f "name=%v" --format {{.ID}}`, containerName)
```

Comment:
@agajdosi I guess the name is a filter , so there is a possibility that it can return multiple container names, isn't it?

Reply:
@LalatenduMohanty It is possible, there could be 2 containers matching the name, however I do not expect that in our case, would be a really major bug. I was thinking about it before and went without a check for this case, however I can add it.